core/src/modifiers/actions.rs

```diff
@@ -506,130 +423,91 @@ impl<V: View> View for Geo<V> {
     }
 }
 
-/// Methods to add an event handler to a view.
-///
-/// This trait is part of the prelude.
-pub trait Actions<'a> {
-    type View: View;
-    fn on_press<F>(self, action: F) -> Handle<'a, Press<Self::View>>
-    where
-        F: 'static + Fn(&mut Context);
-
-    fn on_release<F>(self, action: F) -> Handle<'a, Release<Self::View>>
+/// Modifiers which add an action callback to a view.
+pub trait ActionModifiers<'a, V: View> {
+    /// Adds a callback which is performed when the view is pressed on with the left mouse button.
+    fn on_press<F>(self, action: F) -> Handle<'a, Press<V>>
     where
         F: 'static + Fn(&mut Context);
 
-    fn on_hover<F>(self, action: F) -> Handle<'a, Hover<Self::View>>
+    /// Adds a callback which is performed when the left mouse button is released on a view after being pressed.
+    fn on_release<F>(self, action: F) -> Handle<'a, Release<V>>
     where
         F: 'static + Fn(&mut Context);
 
-    fn on_over<F>(self, action: F) -> Handle<'a, Over<Self::View>>
+    /// Adds a callback which is performed when the mouse pointer moves over or away from a view.
+    fn on_hover<F>(self, action: F) -> Handle<'a, Hover<V>>
```

Comment:
I'm generally not in favor of multiplexing interfaces like this. How about on_hover_in, on_hover_out, etc?

Reply:
If there's precedent for this kind of interface in other frameworks I could be swayed.

Reply:
The inspiration is from siftui: <https://developer.apple.com/documentation/swiftui/view/onhover(perform:)>. But I think I agree that this isn't necessarily better, since it now requires an `if` statement every time you want to do something on hover. I'd like to keep the names as simple as possible, so how about `on_hover` vs `on_leave`?

Reply:
The problem with on_hover and on_leave is it's more things to memorize or look up. It would be okay if it was hover/leave, but we would need to come up with good names for hover in/out, over in/out, and focus in/out.

Reply:
You make a good point. I think one reason to maybe keep it as `on_hover`, `on_over` and `on_focus` is to match the corresponding css pseudoclass selectors. But then if we have `on_hover_out`, `on_focus_out` and `on_over_out` would this be more confusing because of the asymmetry?

Reply:
I think the confusingness is mitigated by the fact that when you do a docs search for the css selector, the right things come up :)